tests/unit_tests/hooks/test_rest_api.py

```diff
@@ -108,22 +106,20 @@ def test_create_run(self, test_default_conn) -> None:
         """
         test_run: Run = helpers.run_factory.build()
         url = f"https://{test_default_conn.host}/runs"
-        create_payload = CreateRunPayload.create(
-            name=test_run.name,
-            runtime=Runtime.SEDONA,
-            python=PythonRunPayload(
-                uri="s3://bucket/test.py",
-                args=["arg1", "arg2"],
-                entrypoint="src.main",
-            ),
-        )
+        create_payload = {
+            "name": test_run.name,
+            "runtime": Runtime.SEDONA.value,
```

Comment:
`.value` is redundant ?

Reply:
I extract the string value out so that the payloads printed won't include the Runtime class info